DEVELOPING.md

```diff
@@ -12,23 +12,20 @@ Our development setup assumes a LINUX/BSD environemnt.
 6. Add the upstream repo to your remotes `git remote add upstream git@github.com:box/box-ui-elements.git`.
 7. Verify your remotes are properly set up `git remote -v`. You should pull updates from the `upstream` box repo and push changes to your fork `origin`.
 
-## Examples Page (aka the Style Guide)
+## Storybook
 
-A published version of the style guide can be seen at https://opensource.box.com/box-ui-elements/. The style guide uses **_live_** demo data within the UI Elements. Due to security restrictions this data is read-only. You can run a local version of the style guide as follows:
+We use Storybook (https://storybook.js.org/). A published version of our Storybook is available at https://opensource.box.com/box-ui-elements/.
 
-1. Start the style guide server via `yarn start`.
-2. Navigate to `http://localhost:6060/` to see the UI Elements in action. If testing on a different machine or VM, you can instead use the IP address shown on your terminal window.
+## Local development
 
-**_NOTE:_** _This note applies to testing top level UI Elements only and not the lower level components. If you want to use your own live data for the style guide, then start the style guide server using [your own developer auth token](https://developer.box.com/docs/authenticate-with-developer-token) and a file or folder ID via_
-
-- `TOKEN=<YOUR_TOKEN> FILEID=<YOUR_FILE_ID> yarn start` or
-- `TOKEN=<YOUR_TOKEN> FOLDERID=<YOUR_FOLDER_ID> yarn start`
+1. Start your local Storybook instance via `yarn start`.
+2. Navigate to `http://localhost:6061/` to see the UI Elements in action. If testing on a different machine or VM, you can instead use the IP address shown on your terminal window.
```

Comment:
can we update the storybook command to open on port 6060? storybook was set to 6061 since styleguide was using 6060

Reply:
on it!